recipes/napari-plugin-manager/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "napari-plugin-manager" %}
+{% set version = "0.1.0a2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-plugin-manager-{{ version }}.tar.gz
+  sha256: 70de69b197eed9ad61087d437af7c7a282b9e8521072e5588f2775f15537ad1e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=42
+    - wheel
+    - setuptools-scm >=3.4
+    - pip
+  run:
+    - python >=3.8
+    - npe2
+    - qtpy
+    - superqt
+    - pip
+    # - napari >=0.4.18
+
+test:
+  imports:
+    - napari_plugin_manager
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/napari/napari-plugin-manager
+  summary: Install plugins for napari, in napari.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jaimergp
+    - goanpeca
```

Comment:
@goanpeca , are you ok being a maintainer here?

Reply:
Yes I m ok 👍🏼